vspace/src/main/java/edu/asu/diging/vspace/core/model/impl/Space.java

```diff
@@ -35,10 +35,13 @@
         strategy = "edu.asu.diging.vspace.core.data.IdGenerator")
     private String id;
 
-    private SpaceStatus spaceStatus;
+    private SpaceStatus spaceStatus; //-> PUBLISHED / UNPUBLISHED
     @Access(AccessType.PROPERTY)
-    private boolean showUnpublishedLinks;
-
+    private boolean showUnpublishedLinks; // -> true/false
+    
+    @Access(AccessType.PROPERTY)
+    private boolean hideLinksToUnpublishedSpaces;
```

Comment:
the this boolean should indicate if all links to this space should be hidden, not to hide links to unpublished spaces (that's what the previous boolean is for)